cpp/include/cudf/search.hpp

```diff
@@ -57,12 +57,12 @@ namespace cudf {
  *   result   = {   3 }
  * @endcode
  *
- * @param haystack The table containing search space
- * @param needles Values for which to find the insert locations in the search space
- * @param column_order Vector of column sort order
- * @param null_precedence Vector of null_precedence enums needles
- * @param mr Device memory resource used to allocate the returned column's device memory
- * @return A non-nullable column of cudf::size_type elements containing the insertion points
+ * @param haystack The table containing search space.
```

Comment:
Following the documentation guidelines below, periods are not needed for parameter descriptions. The general rule is that complete sentences need periods, but sentence fragments (like the content of a bulleted list or these parameter descriptions) do not need a period.

https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/DOCUMENTATION.md

I believe that means the changes in this file can all be reverted.